client/additional-methods-setup/methods-selector/add-payment-methods-task.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * External dependencies
+ */
+import React, { useCallback, useContext, useState } from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+import {
+	Button,
+	Card,
+	CardBody,
+	CardDivider,
+	CheckboxControl,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import interpolateComponents from 'interpolate-components';
+
+/**
+ * Internal dependencies
+ */
+import WizardTaskContext from '../wizard/task/context';
+import CollapsibleBody from '../wizard/collapsible-body';
+import WizardTaskItem from '../wizard/task-item';
+import PaymentMethodCheckboxes from '../../components/payment-methods-checkboxes';
+import PaymentMethodCheckbox from '../../components/payment-methods-checkboxes/payment-method-checkbox';
```

Comment:
Should we use absolute paths for imported components? I've always preferred absolute paths because if you need to refactor, you don't have to rewrite the paths, but I'd like to know your opinion on that.

Reply:
I'm instead of the opposite view (in this specific case/plugin) because it gets hard for me to know where imports are coming from as I'm reading the code.
So in this case, I would have to change (for example) the import to `import WizardTaskContext from 'additional-methods-setup/wizard/task/context';`. But `additional-methods-setup` is not prefixed by anything. So is `additional-methods-setup` a package? No, it's a directory in the plugin. But how would I know that? I would need to know all the directory names in `client/*`.
I would feel different in this case if we had a prefix on the package names, like `woocommerce-payments/*` (in this case it could be `woocommerce-payments/additional-methods-setup/wizard/task/context`). I would have configured webpack with the `resolve.alias` configuration :/ 
So I changed the import to be a relative path because at least I know (and I hope I can communicate) that the components are "internal"/within the plugin, and not external packages.

Reply:
ok, sounds good! I was just mentioning it because I've had bad experiences with relative paths in the past. In my mind, I always see all imports as packages, whether they're local or not, but I get that's totally my way of doing things.

Reply:
Oh, forgot to mention, that the possible confusion between global packages and local files I think it' solved by the separation we make in all JS files between global imports (on top) and local imports (below). But again, as I said, just my point of view and apparently everyone else in the project uses relative paths. Thanks for sharing your point of view!

Reply:
Thank _you_! :) 
Like you mentioned, since the project uses a mix of relative and absolute imports for "local" files, I'll keep it as is - it can always be changed by the next person modifying these files 😆